packages/block-library/src/post-author/index.php

```diff
@@ -29,6 +29,9 @@ function render_block_core_post_author( $attributes, $content, $block ) {
 		$attributes['avatarSize']
 	) : null;
 
+	$link        = ! empty( $attributes['linked'] ) ? get_author_posts_url( $author_id ) : '';
+	$author_name = $link ? '<a href="' . $link . '">' . get_the_author_meta( 'display_name', $author_id ) . '</a>' : get_the_author_meta( 'display_name', $author_id );
+	
```

Comment:
This part is working fine, but, maybe, in order to keep consistency with other blocks, we could do:
1) Define the default author name: `$author_name = get_the_author_meta( 'display_name', $author_id );`
2) Update the value if the link attribute is enabled:
```
if ( ! empty( $attributes['isLink'] ) && ! empty( $attributes['linkTarget'] ) ) {
		$comment_author = sprintf( '<a rel="external nofollow ugc" href="%1s" target="%2s" >%3s</a>', esc_url( $link ), esc_attr( $attributes['linkTarget'] ), $comment_author );
	}
```
3) Check if rel attribute is needed, add a target option.

Comment Author Name block is an example you can use 😄 
https://github.com/WordPress/gutenberg/blob/ac4d4b06715add08e619e01d527b1a270d114680/packages/block-library/src/comment-author-name/index.php#L37-L39

Reply:
Thanks! I'll update to match the implementation used in  `comment-author-name`.